situp-plugins/elasticsearch/src/test/java/com/amazon/situp/plugins/sink/elasticsearch/ElasticsearchSinkIT.java

```diff
@@ -267,6 +308,17 @@ private Integer getDocumentCount(final String index, final String field, final S
     return mappings;
   }
 
+  private String getIndexPolicyId(final String index) throws IOException {
+    final Request request = new Request(HttpMethod.GET, "/_opendistro/_ism/explain/" + index);
+    final Response response = client().performRequest(request);
+    final String responseBody = EntityUtils.toString(response.getEntity());
+
+    @SuppressWarnings("unchecked")
+    final String policyId = (String) ((Map<String, Object>)createParser(XContentType.JSON.xContent(),
+            responseBody).map().get(index)).get("index.opendistro.index_state_management.policy_id");
+    return policyId;
+  }
+
   private boolean deleteDirectory(final File directoryToBeDeleted) {
```

Comment:
three may be a one liner with Apache commons or Guava

Reply:
actually it does not seem to be called. If it is can this be part of the after method?

Reply:
It is still being called in `testOutputRawSpanWithDLQ` so I will replace